releases/3.0.11.md

```diff
@@ -0,0 +1,54 @@
+---
+title: TiDB 3.0.11 Release Notes
+category: Releases
+---
+
+# TiDB 3.0.11 Release Notes
+
+Release date: March 4, 2020
+
+TiDB version: 3.0.11
+
+TiDB Ansible version: 3.0.11
+
+## TiDB 3.0.11 Compatibility Changes
+
+* TiDB
+    + Add the `max-index-length` configuration item to control the maximum index length, which is compatible with the behavior of TiDB versions before 3.0.7 or of MySQL [#15057](https://github.com/pingcap/tidb/pull/15057)
+
+## New Features
+
+* TiDB
+    + Support showing the meta information of partitioned tables in the `information_schema.PARTITIONS` table [#14849](https://github.com/pingcap/tidb/pull/14849)
+
+* TiKV
+    + Optimize the log output by removing unnecessary logs [#6657](https://github.com/tikv/tikv/pull/6657)
+
+* TiDB Binlog
+    + Support the bidirectional data replication between TiDB clusters [#884](https://github.com/pingcap/tidb-binlog/pull/884) [#909](https://github.com/pingcap/tidb-binlog/pull/909)
+
+* TiDB Lightning
+    + Support the TLS configuration [#44](https://github.com/tikv/importer/pull/44) [#270](https://github.com/pingcap/tidb-lightning/pull/270)
+
+* TiDB Ansible
+    + Modify the logic of `create_users.yml` so that users of the central control machine can be inconsistent with `ansible_user` [#1184](https://github.com/pingcap/tidb-ansible/pull/1184)
```

Comment:
```suggestion
    + Modify the logic of `create_users.yml` so that users of the central control machine do not have to be consistent with `ansible_user` [#1184](https://github.com/pingcap/tidb-ansible/pull/1184)
```